components/cli/pkg/util/utils.go

```diff
@@ -532,7 +533,7 @@ func RenameFile(oldName, newName string) error {
 }
 
 func ReplaceFile(fileToBeReplaced, fileToReplace string) error {
-	errRename := RenameFile(fileToBeReplaced, fileToBeReplaced + "-old")
+	errRename := RenameFile(fileToBeReplaced, fileToBeReplaced+"-old")
```

Comment:
This looks like a formatting issue right? Isn't it required to have the space after '+'?

Reply:
This is the go formatting standard.